example/test.py

```diff
@@ -421,12 +415,12 @@ def test_expected_processes(self):
              '--concurrency 1'],
         ])))
 
-    def test_amqp_queues_created(self):
+    def test_amqp_queues_created(self, amqp_container):
```

Comment:
The Celery containers don't seem to have any interaction with the web containers. I actually have very little understanding of how Django/Celery interact 😞